Example/WalletApp/PresentationLayer/Wallet/AuthRequest/AuthRequestView.swift

```diff
@@ -51,8 +33,145 @@ struct AuthRequestView: View {
                         .lineSpacing(4)
                         .padding(.top, 8)
                     
+                    switch presenter.validationStatus {
+                    case .unknown:
+                        HStack(spacing: 5) {
+                            Image(systemName: "exclamationmark.circle.fill")
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
+                                .foregroundColor(.orange)
+                            
+                            Text("Cannot verify")
+                                .foregroundColor(.orange)
+                                .font(.system(size: 14, weight: .semibold, design: .rounded))
```

Comment:
You can apply these modifiers only once to the entire HStack, removing the duplication.